ifrs17/Import/2ImportScope-PresentValue.ipynb

```diff
@@ -307,9 +318,9 @@
                 "\n{",
                 "\n    [NotVisible]",
                 "\n    string EconomicBasis => GetContext();",
-                "\n    private double[] CurrentValues => GetScope<DiscountedCashflow>(Identity).Values;",
+                "\n    protected double[] CurrentValues => GetScope<DiscountedCashflow>(Identity).Values;",
```

Comment:
can this be private again? maybe i missed the usage in the inherited interface....